lib/installer.js

```diff
@@ -11,9 +11,19 @@ const WAD_DL = `https://github.com/Microsoft/WinAppDriver/releases/download/v${W
 const WAD_DL_MD5 = 'dbaa9a3f7416c2b73cc5cd0e7452c8d0';
 const WAD_TIMEOUT = 5000; //ms
 
-const WAD_INSTALL_PATH = path.resolve(
-  process.env['ProgramFiles(x86)'] || process.env.ProgramFiles || 'C:\\\\Program Files',
-  'Windows Application Driver', 'WinAppDriver.exe');
+// check whether or not the path actually exists
+function resolveDriver (folder) {
+  return path.resolve(folder, 'Windows Application Driver', 'WinAppDriver.exe');
```

Comment:
yes, this is possible and correct. Although I have at least two concerns:
- we hardcode the path to the driver executable. Ideally we should read it from the installer properties in the system registry. Although I cannot find the place where the bundled MSI installer does that, probably because package uninstall is disabled there (?)
- all sync calls actually delay thread execution in an async code. I prefer to use async calls unless there is a really major reason not to do so.